.github/workflows/release.yml

```diff
@@ -0,0 +1,52 @@
+name: Release SDK, devpack, Gradle plugin
+
+on:
+  workflow_dispatch:
+    inputs:
+      snapshot:
+        description: 'Publish snapshot?'
+        required: false
+        default: 'false'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up JDK
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'adopt'
+          java-version: '8'
+          cache: 'gradle'
+      - name: Login to GitHub Container Registry # Required for integration tests
+        uses: docker/login-action@v1
```

Comment:
Can you update it to `docker/login-action@v3`?

https://github.com/docker/login-action

Can you also update this on the `.github/workflows/integration.yml` file?